apps/website-new/docs/en/practice/bridge/react-bridge.mdx

```diff
@@ -1,9 +1,37 @@
-# React Bridge
+# React Bridge（for React v16-v19）
 
 `@module-federation/bridge-react` provides bridge utility functions for React applications:
 - `createBridgeComponent`: Used for exporting application-level modules, suitable for producers to wrap modules exported as application types
 - `createRemoteComponent`: Used for loading application-level modules, suitable for consumers to load modules as application types
 
+### React Version Compatibility
+
+`@module-federation/bridge-react` supports multiple major React versions, ensuring seamless integration between applications using different React versions:
+
+- **React 16/17 (Legacy)**: Fully supported, using traditional `ReactDOM.render` and `ReactDOM.unmountComponentAtNode` APIs
+- **React 18**: Fully supported, using the new `createRoot` and `hydrateRoot` APIs
+- **React 19**: Fully supported, using `createRoot` and `hydrateRoot` APIs, but with legacy render methods completely removed
+
+### Version-Specific Imports
+
+You can choose specific import paths based on the React version used in your project:
+
+```jsx
+// Auto-detect React version (recommended for most scenarios)
```

Comment:
How to detect, I'm not found the logic.(maybe just not in this diff)

Reply:
@KyrieLii at here https://github.com/module-federation/core/pull/3690/files#diff-800ac765ab64aaf3fe6ab696d2418909a1347b3c199cab30aeb9134a01b0aa61R24